PSReadLine/BasicEditing.cs

```diff
@@ -246,6 +246,74 @@ public static void DeleteCharOrExit(ConsoleKeyInfo? key = null, object arg = nul
             _singleton.DeleteCharImpl(1, orExit: true);
         }
 
+        /// <summary>
+        /// Upcase the current word and move to the next one.
+        /// </summary>
+        public static void UpcaseWord(ConsoleKeyInfo? key = null, object arg = null)
```

Comment:
Currently, when run `Get-PSReadLineKeyHandler`, these 3 functions are shown as "User defined functions":

![image](https://user-images.githubusercontent.com/127450/178384911-756da5d2-d00b-4149-8f97-afdb1fbf18e1.png)

To make them be grouped to the "Basic editing functions" section, you need to add them to [GetDisplayGrouping](https://github.com/PowerShell/PSReadLine/blob/bbbe43bde346c443e236ac39fff1103cd3b2ccd0/PSReadLine/KeyBindings.cs#L398). Make sure to keep the alphabetical order.